instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestWrapperBase.java

```diff
@@ -21,9 +21,18 @@
 abstract class TracingRequestWrapperBase<I, O> extends TracingRequestHandler<I, O> {
 
   protected static final ObjectMapper OBJECT_MAPPER =
-      new ObjectMapper()
-          .registerModule(new CustomJodaModule())
-          .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+      new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
`aws-lambda-java-core` doesn't depend on jackson, would this also be an issue?

Reply:
Yeah think so. It looks like the best thing is to go ahead and switch this into lambda core and lambda events instrumentation, otherwise things get too hectic. Will try that instead.

Reply:
Realized jackson is OK since the lambda runtime itself includes jackson, but it doesn't include events or joda.